playbooks/roles/common_vars/defaults/main.yml

```diff
@@ -142,8 +142,7 @@ common_release_specific_debian_pkgs:
   bionic:
     - python-pip
     - python3.5-dev
```

Comment:
Should we also remove it from bionic?

Reply:
It probably wouldn't hurt, but I'm hoping to keep the changes as small as possible.